sound/Music.java

```diff
@@ -203,21 +213,58 @@ public void playTune(Tune tune) {
         while (tuneIt.hasNext()) {
             System.out.println("[Line " + (line++) + "] ");
             chord = tuneIt.next();
-            playChord(chord,duration);
+            playChord(chord, duration);
         }
         silence();
     }
-    
+
     /**
      * Stop all tones coming from synthesizer.
      */
     public void silence() {
         Note note;
         int numNotes = noteHistory.size();
-        for (int i=0; i < numNotes; i++) {
+        for (int i = 0; i < numNotes; i++) {
             note = noteHistory.get(i);
-            channels[note.channel()].noteOff(note.pitch(),note.velocityOn());
+            channels[note.getChannel()].noteOff(note.getPitch(),
+                    note.getVelocityOn());
         }
     }
 
+    /**
+     *@return synthesizer interface
+     *        reference
+     */
+      Synthesizer getSynthesizer() {
+         return synth;
+     }
+
+     /**
+      *@return channels in this synth
+      */
+     MidiChannel[] getChannels() {
+        return channels;
+     }
+
+     /**
+      *@return Midi instrument number
+      */
+     int getInstrument() {
+        return instrument;
+     }
+
+     /**
+      *@return this notes velocity
+      */
+      int getVelocity() {
+         return velocity;
+      }
+
+     /**
+      *@return number of channels available
+      *        for this synth
+      */
+      int getNumChannels() {
+         return numChannels;
```

Comment:
I'm getting confused about parallel PRs that affect the same lines of files.
Please delete all of these lines.
I'm no longer sure what to do with the multiple PRs.
Do you want to mark one of them as DRAFT?  I'll then work on the other one first.

Reply:
I am not sure why that is either. This one takes precedence since it is the correct modifier and is part of the checkstyle resolution.

Reply:
OK.  Given that this PR, #41, is presumed to be the one that is active, I'll close #39.  (I've already set PR #39 to DRAFT, added the Question tag, and moved it into To Do.)